src/WebJobs.Script/Diagnostics/DiagnosticEventConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    internal static class DiagnosticEventConstants
+    {
+        public const string HostIdCollisionErrorCode = "AZFD004";
+        public const string HostIdCollisionHelpLink = "https://go.microsoft.com/fwlink/?linkid=2224100";
+
+        public const string ExternalStartupErrorCode = "AZFD0005";
+        public const string ExternalStartupErrorHelpLink = "https://go.microsoft.com/fwlink/?linkid=2224847";
```

Comment:
this link gives 404. Do we need to setup something somewhere else for this to point to the right resource?

Reply:
PR for this is here -- https://github.com/MicrosoftDocs/azure-docs-pr/pull/227355. Should be released by the time this goes out.